packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -529,4 +530,24 @@ export class MetriportMedicalApi {
     const resp = await this.api.post(url, docRef);
     return resp.data;
   }
+
+  /**
+   * Verifies the signature of a webhook request.
+   * Refer to Metriport's documentation for more details: https://docs.metriport.com/medical-api/more-info/webhooks.
+   *
+   * @param wh_key - your webhook key.
+   * @param cxId - your cxID.
```

Comment:
Only noticed it now, this is not a parameter - and even if it was, `cxID` doesn't mean anything to our users.

No need to block this release, but can you update it on your next PR, please?